docs/readthedocs/config.md

```diff
@@ -70,6 +70,9 @@ simona.input.weather.datasource = {
     - The sample values should only be used to test the functionality. The performance of a reasonable simulation with sensitive results should be based on real weather data.
     - Supported weather data sources are: influxdb1x, csv, sql, couchbase, sample
 
+Further model classes which can be used to describe a data set as input to power system simulations are described in [PSDM](https://powersystemdatamodel.readthedocs.io/en/latest/models/models.html). 
+Data sink structure and data deployment are explained in the [I/O-capabilities](https://powersystemdatamodel.readthedocs.io/en/latest/io/basiciousage.html) section of the PSDM. 
```

Comment:
I wouldn't necessarily call it data deployment... I think you're rather looking at data sources and sinks, maybe?